src/containers/trends/CBGTrendsContainer.js

```diff
@@ -66,14 +65,56 @@ class CBGTrendsContainer extends React.Component {
       targetLowerBound: PropTypes.number.isRequired,
       veryLowThreshold: PropTypes.number.isRequired,
     }),
-    bgUnits: PropTypes.oneOf(['mg/dL', 'mmol/L']),
-    data: PropTypes.array.isRequired,
-    focusedSlice: PropTypes.object,
-    focusedSliceKeys: PropTypes.array,
+    bgUnits: PropTypes.oneOf(['mg/dL', 'mmol/L']).isRequired,
```

Comment:
I've done it now anything, not exactly a massive change to copy over, and I'm afraid I'll forget.